### PR TITLE
Upgrade hamcrest libraries to v1.3

### PR DESCRIPTION
**Pull request by ava-fred**
Upgrade the hamcrest libraries to v1.3 to get rid of the dependency on
the org.hamcrest plugin.
Also, adjust the classpath of the c.a.t.ddk.xtext.format.test project to
remove the errors in its resource folder in the Eclipse Problems view.

### CODE REVIEW

File: ddk-target/ddk.target [hunk 1]
```diff
@@ -1,43 +1,45 @@
...
+    <repository location="https://download.eclipse.org/tools/orbit/downloads/drops/R20200529191137/repository"/>
+    <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.core.source" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
+    <unit id="org.hamcrest.library.source" version="1.3.0.v20180524-2246"/>
+    <unit id="org.mockito" version="2.23.0.v20200310-1642"/>
+    <unit id="org.mockito.source" version="2.23.0.v20200310-1642"/>
+  </location>
+</locations>
+</target>
```
Comment (rubenporras) [inline ddk-target/ddk.target:45]: Hi Fredi,

it looks like we have here change in the end of line character(s), is this wanted or shall we revert it?

File: ddk-target/ddk.target [hunk 2]
```diff
@@ -39,5 +32,14 @@
...
+    <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.core.source" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
+    <unit id="org.hamcrest.library.source" version="1.3.0.v20180524-2246"/>
+    <unit id="org.mockito" version="2.23.0.v20200310-1642"/>
+    <unit id="org.mockito.source" version="2.23.0.v20200310-1642"/>
+  </location>
 </locations>
-</target>
+</target>
```
Comment (rubenporras) [inline ddk-target/ddk.target:45]: Can we put back the removed end of line?

File: ddk-target/ddk.target (hunk 1 above)
Comment (ava-fred) [reply ddk-target/ddk.target]: done

File: ddk-target/ddk.target (hunk 2 above)
Comment (ava-fred) [reply ddk-target/ddk.target]: done

**Review by rubenporras — APPROVED** [review]

**Pull request merged**